src/Calculator.php

```diff
@@ -23,29 +23,41 @@ public function __construct(ComposerFile $composer, PackagistAPI $packagist)
     public function getDependencyInfo(string $directory): array
     {
         $dependencies = $this->composer->getDependencies($directory);
+        $repositories = array_map(
+			fn($repositoryUrl): ?Repository => $this->packageAPI->getRepositoryInfo($repositoryUrl),
+			$this->composer->getRepositoriesUrl($directory) ?: ['https://repo.packagist.org']
```

Comment:
Can you make sure you’re using either tabs or spaces for indentation? I suggest you use spaces since the projects seems to use them already?

Reply:
Done